src/core/pages/AccountInfo.vue

```diff
@@ -156,13 +159,13 @@ export default {
       try {
         this.loading = true;
 
-        this.$v.userProfile.local.password.$touch();
-        this.$v.passwordConfirm.$touch();
-        if (this.$v.userProfile.local.password.$error || this.$v.passwordConfirm.$error) {
+        this.$v.newPassword.password.$touch();
+        this.$v.newPassword.passwordConfirm.$touch();
```

Comment:
tu peux simplifier en this.$v.newPassword.$touch() du coup car en faisant un `touch` sur l'objet tu l'appliques sur chacun des champs

Reply:
en effet vu qu'on fait les deux en même temps là on peut se le permettre